deploy/gk-deploy

```diff
@@ -214,24 +234,26 @@ if [ "x${CLI}" == "x" ]; then
   fi
 fi
 
-if [ "${CLI}" == "oc" ]; then
+CLI="${CLI} -n ${namespace}"
+
+if [ "${CLI}" == *oc* ]; then
   output "Starting OpenShift deployment."
-  oc new-project heketi
+  ${CLI} new-project heketi
   oadm policy add-scc-to-user privileged -z default
-  oc create -f /usr/share/heketi/templates
+  ${CLI} create -f /usr/share/heketi/templates
```

Comment:
@humblec maybe "could be" or "should be", but that is actually a completely independent request, right? We should not require to fix/improve everything we can think of in this patch. :-)